swedish-oidc-claims-specification.md

```diff
@@ -390,104 +388,156 @@ There are basically two ways for a Relying Party to indicate which claims it wis
+potentially, unnecessary call to the UserInfo endpoint.
+
+For each scope defined below, a listing of the claims that the scope value requests (access to) is
+declared. Each scope definition also presents a "claims parameter equivalent", i.e., how the
+claims would be requested using the `claims` request parameter. This tells where claims should be
+delivered (ID token and/or UserInfo endpoint), and whether the claims should be regarded as essential
+or voluntary to deliver by the OP.
+
+> Note that the use of "essential" does not imply that the OP must deliver the claim, only
+that its delivery is marked as essential for the Relying Party's ability to continue the
+specific task requested by the end-user (for example, logging the user in to the RP application,
+or ensuring a smooth authorization for a specific task).
+
+> If a scope definition states that a certain claim should be delivered in the ID token, its definition
+will in many cases also include the same claim for delivery via the UserInfo endpoint. The reason for 
+this is that the UserInfo endpoint should offer a complete set of user identity claims (based on 
+the authorization of the RP).
+
+**Note:** The scope definitions regarding the delivery location of claims assumes that authentication
+requests are made using `response_type=code`. 
+
+<a name="natural-person-information"></a>
+### 3.1. Natural Person Information
+
+**Scope:** `https://id.oidc.se/scope/naturalPersonInfo`
+
+**Description:** A scope that defines a claim set that provides basic natural person information
+normally associated with a Swedish eID.
+
+| Requested Claims | Description/comment | Reference |
+| :--- | :--- | :--- |
+| `family_name` | Surname/family name | \[[OpenID.Core](#openid-core)\] |
+| `given_name` | Given name | \[[OpenID.Core](#openid-core)\] |
+| `middle_name` | Middle name | \[[OpenID.Core](#openid-core)\] |
+| `name` | Display name | \[[OpenID.Core](#openid-core)\] |
+| `birthdate` | Date of birth | \[[OpenID.Core](#openid-core)\] |
+
+**Claims Parameter Equivalent:**
+
+```
+{
+  "userinfo" : {
+    "family_name" : null,
+    "given_name" : null,
+    "middle_name" : null,
+    "name": null,
+    "birthdate" : null
+  }
+}
+```
+
+**Note:**: The `https://id.oidc.se/scope/naturalPersonInfo` scope is a subset of the `profile` standard
+scope as defined in section 5.4 of \[[OpenID.Core](#openid-core)\].
+
+The `profile` scope is pretty much intended as a scope for an Internet user wishing to create an 
+account on a website. Claims  such as `preferred_username`, `picture` and `website` indicates that.
+Of course, not all claims within the scope need to be delivered, but for the sake of privacy a Relying Party should not ask for more claims than it actually requires. Therefore, this specification 
+defines the `https://id.oidc.se/scope/naturalPersonInfo` scope to limit the amount of user identity
+claims to what is offered using a Swedish eID.
 
```

Comment:
Consider removing "pretty much"

Reply:
Should we consider adding gender as a possible return claim?

Reply:
> Should we consider adding gender as a possible return claim?

I think I avoid gender as long as possible. From being a very binary thing it is now something else.